performance-storage-service/requirements.txt

```diff
@@ -12,3 +12,11 @@ pytz==2020.1
 PyYAML==5.3.1
 six==1.15.0
 sqlparse==0.3.1
+asgiref==3.2.7
+Django==3.0.6
```

Comment:
this is duplicate, it occurs twice in this file

Reply:
actually, the following lines are duplicate
```
Django==3.0.6
asgiref==3.2.7
djangorestframework==3.11.0
gunicorn==20.0.4
pytz==2020.1
sqlparse==0.3.1
```

Reply:
to update `requirements.txt`
- make sure to enter the virtualenv first
- then run `pip3 freeze > requirements.txt` (dont use `>>` since that is for appending)

Reply:
ah thats what I have been doing wrong `>>` Thanks.